setup.py

```diff
@@ -21,15 +21,15 @@ def requirements():
         return [pname.strip() for pname in f.readlines()]
 
 setup(name='ragavi',
-      version="0.0.5",
+      version="0.0.6",
```

Comment:
rather make it `0.0.7` since `0.0.6` perished. Since this seems like a major release you can make it `0.1.0` ;)

Reply:
Ok, bumped to version `0.0.7`.